content/blog/2022-07-12-announcing-localstack-v1-general-availability/index.md

```diff
@@ -7,8 +7,11 @@ lastmod: 2022-07-12T11:00:55+05:30
 images: []
```

Comment:
Hi @whummer @Sheg-Ge — Can you help with description and lead, please?

Reply:
Description looks good to me, suggestion for lead:

```
lead: LocalStack 1.0 introduces a whole suite of significant enhancements in the platform, plus exciting new features for team collaboration and stack analytics.
```
Also, let's update the date to `2022-07-13` (the release date) 👍